pkg/monitors/kubernetes/volumes/pvdim.go

```diff
@@ -47,10 +47,13 @@ func findVolumeInPod(pod *v1.Pod, volName string) *v1.Volume {
 }
 
 func dimsForPersistentVolumeSource(pvs v1.PersistentVolumeSource) map[string]string {
-	// IF YOU ADD A NEW TYPE HERE, ADD IT IN dimsForVolumeSource BELOW TOO!
+	// IF YOU ADD A NEW PERSISTENT TYPE HERE, ADD IT IN dimsForVolumeSource BELOW TOO!
 	if pvs.AWSElasticBlockStore != nil {
```

Comment:
Linter is saying that this has enough cases to be a switch.

Reply:
Done.